virt/virt-2-4-release-notes.adoc

```diff
@@ -18,6 +18,9 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 
 //CNV-5371 - Import existing virtual machines from Red Hat Virtualization to OpenShift Virtualization.
 
+//CNV-5439 - TLS certificates
+{VirtProductName} rotates and renews TLS certificates automatically without disrupting operations. All operations including migrations, image uploads, VNC and console connections continue to function without any disruption.
```

Comment:
I think this could be simplified further for the release notes -- I suggest:

`{VirtProductName} rotates and renews TLS certificates at regular intervals. This automatic process does not disrupt any operations.`

Otherwise LGTM. Please squash commits before merge.